source/fundamentals/builders/aggregation-expression-operations.txt

```diff
@@ -919,71 +1133,66 @@ the Query API:
-     - :manual:`$eq </reference/operator/aggregation/eq/>`
+   * - `isArrayOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isArrayOr(com.mongodb.client.model.mql.MqlArray)>`__
+     - *No corresponding operator*
 
-   * - `gt() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#gt(com.mongodb.client.model.mql.MqlValue)>`__
-     - :manual:`$gt </reference/operator/aggregation/gt/>`
+   * - `isBooleanOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isBooleanOr(com.mongodb.client.model.mql.MqlBoolean)>`__
+     - *No corresponding operator*
 
-   * - `gte() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#gte(com.mongodb.client.model.mql.MqlValue)>`__
-     - :manual:`$gte </reference/operator/aggregation/gte/>`
+   * - `isDateOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isDateOr(com.mongodb.client.model.mql.MqlDate)>`__
+     - *No corresponding operator*
 
-   * - `lt() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#lt(com.mongodb.client.model.mql.MqlValue)>`__
-     - :manual:`$lt </reference/operator/aggregation/lt/>`
+   * - `isDocumentOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isDocumentOr(T)>`__
+     - *No corresponding operator*
 
-   * - `lte() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#lte(com.mongodb.client.model.mql.MqlValue)>`__
-     - :manual:`$lte </reference/operator/aggregation/lte/>`
+   * - `isIntegerOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isIntegerOr(com.mongodb.client.model.mql.MqlInteger)>`__
+     - *No corresponding operator*
 
-   * - `ne() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#ne(com.mongodb.client.model.mql.MqlValue)>`__
-     - :manual:`$ne </reference/operator/aggregation/ne/>`
+   * - `isMapOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isMapOr(com.mongodb.client.model.mql.MqlMap)>`__
+     - *No corresponding operator*
 
-.. include:: /includes/fundamentals/agg-mql-diff.rst
+   * - `isNumberOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isNumberOr(com.mongodb.client.model.mql.MqlNumber)>`__
+     - *No corresponding operator*
 
-The following example creates a pipeline stage that matches all the documents
-where the ``location`` field has the value ``"California"``:
+   * - `isStringOr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlValue.html#isStringOr(com.mongodb.client.model.mql.MqlString)>`__
+     - *No corresponding operator*
 
-.. code-block:: java
-   :copyable: true
+Suppose you have a collection of rating data. Early versions of the reviews
+allowed users to submit reviews without a star rating. You want to clean up
```

Comment:
Suggestion:

Seems unnecessary to mention multiple prior versions (plural). Also, it might be helpful to identify what it was a version of.

```suggestion
Suppose you have a collection of rating data. An earlier version of the reviews schema
allowed users to submit reviews without a star rating. You want to clean up
```